documentation/src/main/asciidoc/search-dsl-predicate.asciidoc

```diff
@@ -549,6 +550,24 @@ and *another* of which has a last name matching `asimov`,
 will *not* match.
 ====
 
+[[search-dsl-predicate-nested-implicit]]
+.Use the implicit nested form
+====
+[source, JAVA, indent=0, subs="+callouts"]
+----
+include::{sourcedir}/org/hibernate/search/documentation/search/predicate/PredicateDslIT.java[tags=nested-implicit-form]
+----
+<1> The nested predicate is created implicitly, since target fields here belong to a nested object.
+<2> The author must have a first name matching `isaac`.
+<3> The author must have a last name matching `asimov`.
+<4> All returned hits will be books for which at least one author
+has a first name matching `isaac` and a last name matching `asimov`.
+Books that happen to have multiple authors,
+one of which has a first name matching `isaac`
+and *another* of which has a last name matching `asimov`,
+will *not* match.
```

Comment:
This is wrong, though. They will match, because we apply the nested predicate separately to each `match` predicate.

Reply:
You're right. Let's update the statement.